terraform/main.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  backend "remote" {
+    organization = "tinkerbell"
+
+    workspaces {
+      name = "github_management"
+    }
+  }
+}
+
+variable "github_token" {
+  description = "GitHub token"
+  type        = string
+}
```

Comment:
GITHUB_TOKEN is already looked for by the provider by default so we can just get rid of this variable all together

Reply:
If that's the case, then we need to remove https://github.com/tinkerbell/.github/pull/9/files#diff-2e617c7870fa918457b1eee1c7d67ba82f19d043ae7b7918db26873e18793028R12 too

Reply:
ah yes. I thought I had added it to line-range when making my comment.

Reply:
Does the original value `token = ""` work? (does the provider use empty values as a signal to use the GITHUB_TOKEN variable?)

Reply:
No I fixed that few days ago when you left the comment :) it was a leftover from my side

Reply:
If I remove the variale declaration `terraform validate` fails! As @displague suggested :D I had to add it back